backend/connector-integration/src/connectors/globalpay/transformers.rs

```diff
@@ -310,11 +311,9 @@ pub struct GlobalpayPaymentsRequest<T: PaymentMethodDataTypes> {
     pub country: common_enums::CountryAlpha2,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub capture_mode: Option<GlobalpayCaptureMode>,
-    #[serde(skip_serializing_if = "Option::is_none")]
     pub initiator: Option<Initiator>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub notifications: Option<GlobalpayNotifications>,
-    #[serde(skip_serializing_if = "Option::is_none")]
     pub stored_credential: Option<StoredCredential>,
```

Comment:
Same concern as `initiator` - removing `skip_serializing_if` means `stored_credential: null` will be sent even for non-mandate payments. Verify API compatibility.

If the API expects the field to be omitted when not present:

```suggestion
    #[serde(skip_serializing_if = "Option::is_none")]
    pub stored_credential: Option<StoredCredential>,
```